### ISSUE TITLE
"Play Sound" button only works the first time it's used

### ISSUE THREAD

**Issue by vedxyz** [vedxyz/kana#4]
Likely because a new `Audio` instance isn't being created on each click? To be investigated.

**Comment by zDEFz** [comment]
should be a checkbox instead of a button as a suggestion.

**Comment by vedxyz** [comment]
Hmm, I don't quite get what you mean by that. It's currently a button since it is supposed to play the sound only upon clicking it. A checkbox would imply playing in some automated way, I would think, since it represents a "toggle" then.

**Comment by zDEFz** [comment]
Yes. Thats right. By above issue I understood that the goal was that its always playing back the sound. But apparently that is not the case. But as for my rationale, whats the purpose of having a middle-ground of on and off? As for me, I'd like to have the option to let it always play. Or to have the stroke order always show. But perhaps we can have both.

Currently, as a workaround, I use a userscript. And its far from perfect, it's the first time I created one. And at the time, I use it for https://djtguide.neocities.org/kana/ which is pretty similar.

```// ==UserScript==
// @name         show kana strokeorder on https://djtguide.neocities.org/kana/
// @namespace    http://tampermonkey.net/
// @version      0.1
// @description  Clicks the kana button to always show the strokeorder
// @author       You
// @match        https://*/*
// @match        http://*/*
// @grant        none
// ==/UserScript==

(function() {
    'use strict';

 document.getElementById('input_box').addEventListener('input', function() {
        document.getElementById('tool_stroke').click();
 });
})();```
